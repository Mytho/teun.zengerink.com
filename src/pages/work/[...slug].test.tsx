import React from 'react'
import renderer from 'react-test-renderer'
import mockProjects from '../../__mocks__/projects'
import Work from './[...slug].page'

const defaultProps = { projects: mockProjects }

describe('Work', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Work {...defaultProps} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
