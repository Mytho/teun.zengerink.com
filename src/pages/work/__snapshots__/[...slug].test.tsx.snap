// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Work renders correctly 1`] = `
<div>
  <div>
    <aside
      class="menu"
      data-testid="menu"
    >
      <a
        href="/"
      >
        <h1>
          Teun Zengerink
        </h1>
      </a>
      <div
        class="hamburger"
        data-testid="hamburger"
      >
        <span
          class="line"
        />
        <span
          class="line"
        />
        <span
          class="line"
        />
      </div>
      <nav
        class="navigation"
      >
        <h2>
          Projects
        </h2>
        <ul>
          <li
            class=""
          >
            <a
              href="/work/my-first-project/01"
            >
              My First Project
            </a>
          </li>
          <li
            class=""
          >
            <a
              href="/work/no-statement/01"
            >
              Project without a statement
            </a>
          </li>
        </ul>
        <span
          class="socials"
        >
          <a
            href="https://instagram.com/t.zengerink/"
          >
            <img
              alt="Instagram"
              src="/img/instagram.jpg"
            />
          </a>
        </span>
      </nav>
    </aside>
    <main
      class="main"
    >
      <div
        class="slideshow"
      >
        <div
          class="previous"
          data-testid="left-side"
        />
        <div
          class="next"
          data-testid="right-side"
        />
      </div>
    </main>
  </div>
</div>
`;
